src/utils/lineData.ts

```diff
@@ -51,7 +51,7 @@ export const simplifyLineData = (
     const [simplifedXs, simplifiedYs] = simplify(
       xs.map(x => xScale(x)),
       ys.map(y => yScale(y)),
-      1
+      0.5
```

Comment:
I'm curious what the effect of changing the epsilon here was.

Reply:
[Check it out](https://observablehq.com/@chnn/running-ramer-douglas-peucker-on-typed-arrays) :)